ui_components/src/village/WarTable.js

```diff
@@ -121,6 +127,183 @@ export function WarTable({
+                        </div>
+                    </div>
+                </div>
+                <div className="village_warlog_details">
+                    <div className="village_warlog_details_item">
+                        <div className="village_warlog_details_label">PVP WINS</div>
+                        <div className="village_warlog_details_value">{log.pvp_wins}</div>
+                    </div>
+                    <div className="village_warlog_details_item">
+                        <div className="village_warlog_details_label">RAID</div>
+                        <div className="village_warlog_details_value">
+                            <span>{log.raid_count}</span>
+                            <span className="warlog_red">({log.damage_dealt})</span>
+                        </div>
+                    </div>
+                    <div className="village_warlog_details_item">
+                        <div className="village_warlog_details_label">REINFORCE</div>
+                        <div className="village_warlog_details_value">
+                            <span>{log.reinforce_count}</span>
+                            <span className="warlog_green">({log.damage_healed})</span>
+                        </div>
+                    </div>
+                    <div className="village_warlog_details_item">
+                        <div className="village_warlog_details_label">INFILTRATE</div>
+                        <div className="village_warlog_details_value">{log.infiltrate_count}</div>
+                    </div>
+                    <div className="village_warlog_details_item">
+                        <div className="village_warlog_details_label">DEF</div>
+                        <div className="village_warlog_details_value">
+                            <span className="warlog_green">+{log.defense_gained}</span>
+                            <span className="warlog_red">-{log.defense_reduced}</span>
+                        </div>
+                    </div>
+                    <div className="village_warlog_details_item">
+                        <div className="village_warlog_details_label">CAPTURES</div>
+                        <div className="village_warlog_details_value">{log.regions_captured + log.villages_captured}</div>
+                    </div>
+                    <div className="village_warlog_details_item">
+                        <div className="village_warlog_details_label">PATROLS</div>
+                        <div className="village_warlog_details_value">{log.patrols_defeated}</div>
+                    </div>
+                    <div className="village_warlog_details_item">
+                        <div className="village_warlog_details_label">RESOURCES</div>
+                        <div className="village_warlog_details_value">{log.resources_stolen}</div>
+                    </div>
+                </div>
+            </div>
+        );
+    }
```

Comment:
Can you move these components down below the function body of `WarTable`? Will help with reading through what WarTable itself is doing

Reply:
Done.